### DISCUSSION TITLE
Sliderticks do not play sound

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#25245] [Q&A] [answered]
I've confirmed that this is not a regression from last release, though I don't know if this has just always been the case.
Skin: https://osu.ppy.sh/community/forums/topics/1792308?n=1 (it happens on default skins too)
Map: https://lazer.ppy.sh/beatmapsets/601135#osu/1343792

https://github.com/ppy/osu/assets/53081750/1232e392-4b09-4c69-9abf-ea47b56d3f5d

https://github.com/ppy/osu/assets/53081750/c8a26607-4dcc-4646-9154-ca441a58f8a5

[runtime.log](https://github.com/ppy/osu/files/13179518/runtime.log)

**Comment by peppy** [comment]
I'll take a look into this one.

**Reply by peppy** [reply, marked as answer]
This beatmap has a control point at 24,407 ms with 5% volume:

```
18408,307.692307692308,4,2,7,60,1,0
24407,-100,4,2,7,5,0,0
25792,-100,4,2,7,60,0,0
```

The slider in question ends at ~24,408ms:

```chsarp
  hitObject.GetEndTime()
  24407.923076923078
```

![CleanShot 2023-10-27 at 07 20 37](https://github.com/ppy/osu/assets/191335/97299845-83f6-4468-ac57-30ba9ac3fed9)

We add a leniency of 5 ms to the end time to get the relevant sample point:

https://github.com/ppy/osu/blob/c89081589ccbaabef181530a203300116dd3ea00/osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs#L118

so to lazer's eye, it is correct to apply the 5% adjust here.

I believe this falls under the same umbrella as https://github.com/ppy/osu/issues/24997, although manifesting as a volume discrepancy rather than incorrect sample set.
